plotly_cheatsheet.Rmd

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
Rmd should not contain `---` lines, and should start with a `#` followed by a space and your chapter name.

Reply:
Hello, I uploaded the latest version. Do you have time to take a look?